src/packages/SpeakClient.ts

```diff
@@ -0,0 +1,32 @@
+import { AbstractClient } from "./AbstractClient";
+import { SpeakLiveClient } from "./SpeakLiveClient";
+import { SpeakRestClient } from "./SpeakRestClient";
+import { SpeakSchema } from "../lib/types";
+
+/**
+ * The `SpeakClient` class extends the `AbstractClient` class and provides access to the "speak" namespace.
+ * It exposes two methods:
+ *
+ * 1. `request()`: Returns a `SpeakRestClient` instance for interacting with the rest speak API.
+ * 2. `live(ttsOptions: SpeakSchema = {}, endpoint = ":version/speak")`: Returns a `SpeakLiveClient` instance for interacting with the live speak API, with the provided TTS options and endpoint.
+ */
+export class SpeakClient extends AbstractClient {
+  public namespace: string = "speak";
+
+  /**
+   * Returns a `SpeakRestClient` instance for interacting with the rest speak API.
+   */
+  get request() {
+    return new SpeakRestClient(this.options);
```

Comment:
This is good, I assume to provide backwards compatibility with the dot notation we set up for TTS REST

Reply:
however I think for this one, there is a slight gotcha with how we initially set it up, and how to maintain the compatibility of it.

If you make `get request(){}` look like this...

```
  get request(
    source: TextSource,
    options?: SpeakSchema,
    endpoint = ":version/speak"
  ) {
    const client = new SpeakRestClient(this.options);
 
    return client.request(source, options, endpoint);
  }
```

it _should_ maintain the interface that exists now to do `deepgram.speak.request(...etc)`, despite adding a class between so we can also do `deepgram.speak.live(...etc)`